### PR TITLE
Create KunalVerma.json

### PR DESCRIPTION
**Pull request by kunal00000**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3819"></a>

### CODE REVIEW

File: data/KunalVerma.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      "name": "Twitter",
+      "url": "https://twitter.com/KunalVerma2468",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/kunaahl",
+      "icon": "FaInstagram"
+    }
+  ],
```
Comment (Aadarsh805) [inline data/KunalVerma.json:40]: remove this comma at line 40

File: data/KunalVerma.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**Open Source** | Frontend WebDeveloper",
+  "tags": ["Open Source", "MERN", "Tailwind", "Next Js", "MySQL"],
+  "avatar": "https://github.com/kunal00000.png",
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/KunalVerma2468" },
+    { "icon": "FaGithub", "url": "https://github.com/kunal00000" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/kunalverma2468" },
+    { "icon": "SiHashnode", "url": "https://www.linkedin.com/in/kunalverma2468" },
```
Comment (Aadarsh805) [inline data/KunalVerma.json:12]: remove this comma at line 12

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
change file name to `data/kunal00000.json`

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**